app/javascript/components/admin/AuditModal.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
-import { Button, Modal } from 'react-bootstrap';
 import { PropTypes } from 'prop-types';
-import CustomTable from '../layout/CustomTable';
-import reportError from '../util/ReportError';
+import { Button, Modal } from 'react-bootstrap';
+
+import { formatTimestamp } from '../../lib/DateTimeUtils';
 import axios from 'axios';
-import moment from 'moment-timezone';
 import _ from 'lodash';
 
+import CustomTable from '../layout/CustomTable';
+import reportError from '../util/ReportError';
+
```

Comment:
Only added `import { formatTimestamp } from '../../lib/DateTimeUtils';`. Everything else here is just moving `imports` around.